src/frontends/paddle/src/op/unique.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs unique(const NodeContext& node) {
+    auto x = node.get_input("X");
+
+    std::vector<Output<Node>> outputs;
+    NamedOutputs named_outputs;
```

Comment:
As far as I know, the `NamedOutputs` is a `std::map`. 
In this case, you'd better use constructor with initailization list insteading of inserting k-v one by one, I think.

Reply:
Done